Automattic/vip-go-mu-plugins
# Hiding newer Parse.ly features on silent mode

## Description

In this PR, we are updating the function that hides some of the wp-parsely features when running on silent mode. That is, hiding some parts of the UI in wp-admin when the plugin is enabled through a DB option.

## Changelog Description

### Hiding newer Parse.ly features on silent mode

Updating silent mode to be compatible with newer wp-parsely versions.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Enable wp-parsely through the option.
3. Go to `wp-admin` > `Tools` and see that no Parse.ly settings page appear.
